src/App.vue

```diff
@@ -1,23 +1,24 @@
 <template>
   <div id="app">
-    <img src="./assets/logo.png">
+    <TheHeader></TheHeader>
```

Comment:
multi word components should be registered with dashes `<my-component>`

Reply:
The style guide gives the option of PascalCase or kebab-case, do you have a preference?

https://vuejs.org/v2/style-guide/#Single-file-component-filename-casing-strongly-recommended

Reply:
Ok, pascal case is fine. Most examples I was seeing were kebab. Should rename components to `BaseHeader` and `BaseFooter` though: https://vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended